osmUtils/utils_map.py

```diff
@@ -0,0 +1,63 @@
+"""General utils function to map the retrieved data with folium"""
+import folium
+import json
+from .utils_geo import set_crs
+from shapely.geometry import box
+from .settings import DEFAULT_ZOOM_START, DEFAULT_BASEMAP, DEFAULT_COLOR
+
+
+def generate_folium_map(gdf, kwargs):
+    """
+    Visualize geopandas.GeoDataFrame with folium.
+    
+    Parameters
+    ----------
+    gdf: geopandas.GeoDataFrame
+        GeoDataFrame to be visualized with folium
+
+    **kwargs
+    ---------
+    zoom_start: int
+        Initial zoom level for the map. If Nne, default level set to 10.
+    basemap: string
+        Map tileset to use. If None, cartodb dark_matter used.
+    color: string
+        stroke color. If None, '#f7f5b5' is used.
+    Returns
+    -------
+    folium_map : folium.folium.Map
+    """
+    gdf_projected = set_crs(gdf=gdf, crs='EPSG:3857')
+    gjson = get_gjson(gdf_projected)
+
+    bounds = bounds = list(gdf.bounds.iloc[0])
+    geom = box(bounds[0], bounds[1], bounds[2], bounds[3])
+
+
+    zoom_start=kwargs['zoom_start'] if 'zoom_start' in kwargs else DEFAULT_ZOOM_START
+    basemap=kwargs['basemap'] if 'basemap' in kwargs else DEFAULT_BASEMAP
+    color=kwargs['color'] if 'color' in kwargs else DEFAULT_COLOR
```

Comment:
A more concise way would be:

`zoom_start = kwargs.get('zoom_start', DEFAULT_ZOOM_START)` etc